frontend/src/metabase/ui/components/inputs/Select/SelectItem/SelectItem.module.css

```diff
@@ -4,17 +4,17 @@
   word-break: break-all;
 
   &:hover {
-    color: var(--mb-color-brand);
-    background-color: var(--mb-color-brand-lighter);
+    color: var(--mb-color-text-selected);
+    background-color: var(--mb-color-background-selected);
   }
 
   &.selected {
     color: var(--mb-color-text-white);
-    background-color: var(--mb-color-brand);
+    background-color: var(--mb-color-text-brand);
```

Comment:
Using `text-brand` for a `background-color` for `&.selected` seems a bit strange to me, especially when the above uses `background-selected` for `&.hover` - is this intended? I wonder if this warrants another semantic color name.

Reply:
that might be a mistake, let me double-check. Good catch.

Reply:
Ok, I made a mess. I mistakenly commented on the wrong comment, here's [the comment](https://github.com/metabase/metabase/pull/46365#discussion_r1700336678) for this problem.